src/convert.jl

```diff
@@ -1225,3 +1225,19 @@ function convert(::Type{Hyperplane}, P::HPolyhedron; skip_check::Bool=false)
     c1 = @inbounds first(constraints_list(P))
     return Hyperplane(c1.a, c1.b)
 end
+
+
+"""
+    convert(::Type{SimpleSparsePolynomialZonotope}, P::Zonotope)
+
+convert a zonotope to a simple sparse polynomial zonotope.
+
+"""
+function convert(::Type{SimpleSparsePolynomialZonotope}, P::Zonotope)
+    c = center(P)
+    G = genmat(P)
+    n = ngens(P)
+    E = 1 * Matrix(I, n, n)
```

Comment:
```suggestion
    E = Matrix(1 * I, n, n)
```
This is more efficient because it does not allocate a `Bool` matrix.

```julia
julia> @time 1 * Matrix(I, n, n)
  0.000013 seconds (2 allocations: 160 bytes)
2×2 Matrix{Int64}:
 1  0
 0  1

julia> @time Matrix(1 * I, n, n)
  0.000008 seconds (1 allocation: 96 bytes)
2×2 Matrix{Int64}:
 1  0
 0  1
``